tensorflow_datasets/image/cars196.py

```diff
@@ -79,100 +78,102 @@
 """
 
 class Cars196(tfds.core.GeneratorBasedBuilder):
-  """Car Images dataset"""
+    """Car Images dataset"""
 
-  VERSION = tfds.core.Version('1.0.0')
+    VERSION = tfds.core.Version('1.0.0')
 
 
-  def _info(self):
-     return tfds.core.DatasetInfo(
-        builder=self,
-  
-        description=(_DESCRIPTION),
-        
-        features=tfds.features.FeaturesDict({
-            "image":tfds.features.Image(),
-            "label": tfds.features.ClassLabel(names = _NAMES),
-            "bbox":tfds.features.BBoxFeature(),
-        }),
+    def _info(self):
```

Comment:
Indentation should be 2 as in other files.

Reply:
@us  I messed up my PRs as my first PR was through master and now all other PRs have those changes,what do you suggest me to do?

Reply:
You can create new clean branch from master and then cherry picking this commits.
Like that: `git cherry-pick 12429d1..2478033`
I assume: 
12429d1 : first commit 
2478033 : last commit

Reply:
On the other hand your commits are too messy and too much to adding dataset. Also your commit messages should be more meaningful. When looking at commits, that's not good look and not understandable.